libs/core/algorithms/include/hpx/parallel/task_group.hpp

```diff
@@ -55,7 +58,7 @@ namespace hpx { namespace execution { namespace experimental {
         };
 
     public:
-        // Spawns a task to compute f() and returns immediately.
+        /// \brief Adds a task to compute \c f() and returns immediately.
```

Comment:
Could you add documentation for the parameters as well, please?

Reply:
Done!